.github/workflows/build-ffmpeg.yml

```diff
@@ -253,13 +268,20 @@ jobs:
           echo "::endgroup::"
 
           echo "::group::configure extra flags for cross compilation"
+          extra_configure=""
+          if [[ ${{ matrix.os_type }} == "linux" && ${{ matrix.arch }} == "aarch64" ]]; then
           extra_configure=$(cat <<VAREOF
```

Comment:
For all the heredocs, can you change `<<` to `<<-`? This will allow it to be indented within the if statements.

Then for all if/else statements, can you indent everything within by 2 spaces?

Reply:
Then within the heredoc, another 2 spaces for everything before the last `)`